doc/nrf/doc_styleguide.rst

```diff
@@ -15,33 +15,91 @@ RST style guide
+| Sending shell commands      | Sending Shell Commands           |
++-----------------------------+----------------------------------+
+
+Do not use consecutive headings without intervening text.
+
+Consecutive line rule
+=====================
+
+For readability reasons, start every sentence on a new line in the source files and do not add line breaks within the sentence.
+In the output, consecutive lines without blank lines in between are combined into one paragraph.
 
     .. note:: For the conceptual documentation written in RST, you can have more than 80 characters per line.
               The requirement for 80 characters per line applies only to the code documentation written in doxygen.
 
-Hyperlinks
-==========
-
-All external links must be defined in the ``links.txt`` file.
-Do not define them directly in the RST file.
-The reason for this is to allow for validating all links in one central location and to make it easy to update breaking links.
+Content highlighting
+====================
 
-Each link should be defined only once in ``links.txt``.
+Use the following highlighing rules in the RST conceptual documentation:
 
-If the link text that is defined in ``links.txt`` does not fit in the context where you use the link, you can override it by using the following syntax::
+.. content_highlighting_table:
 
-   `new link text <original link text_>`_
++------------------------+------------------------+---------------------------------------------------------+
+| Markup                 | Example                | Usage criteria                                          |
++========================+========================+=========================================================+
+| ``*emphasis*``         | *relaying*             | Emphasis or new terms.                                  |
++------------------------+------------------------+---------------------------------------------------------+
+| ````code````           | ``west update``        | Code within text, including file paths and names.       |
++------------------------+------------------------+---------------------------------------------------------+
+| ``**PCB**``            | **SW3**                | PCB names.                                              |
++------------------------+------------------------+---------------------------------------------------------+
+| ``:guilabel:`button``` | :guilabel:`Cancel`     | Clickable UI buttons.                                   |
++------------------------+------------------------+---------------------------------------------------------+
 
-It is also possible to define more than one default link text for a link, which can be useful if you frequently need a different link text::
+Do not use the following markup: ```..```.
 
-   .. _`Link text one`:
-   .. _`Link text two`: http://..
+For readability reasons, do not use any highlighting for titles of other pages or headings when they are mentioned in the text.
```

Comment:
I don't agree with that - if we have to link to a Zephyr section that does not have an ID to link to, we use standard citing: "See *Section name* in :ref:`link`"
I think what you wanted to state here is that we shouldn't use highlighting for library names as used in the code - which means that if we get input like "The ``bl_crypto`` library ..." we change it to "The bootloader crypto library ...". At least that's what I wanted to know when we discussed this. ;)